skcosmo/skcosmo_preprocessing/KernelCentering.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
```

Comment:
this file needs to live in `skcosmo/preprocessing/`, and there should be a `skcosmo/preprocessing/__init__.py` file to allow importing this one

Reply:
Done